docs/monitors/ecs-metadata.md

```diff
@@ -0,0 +1,117 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# ecs-metadata
+
+ This monitor reads container stats from a
+[ECS Task Metadata Endpoint version 2](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint-v2.html).
+
+This currently does not support CPU share/quota metrics.
+
+
+Monitor Type: `ecs-metadata`
+
+[Monitor Source Code](https://github.com/signalfx/signalfx-agent/tree/master/internal/monitors/ecs)
+
+**Accepts Endpoints**: No
+
+**Multiple Instances Allowed**: Yes
+
+## Configuration
+
+| Config option | Required | Type | Description |
+| --- | --- | --- | --- |
+| `metadataEndpoint` | no | `string` | The URL of the ECS task metadata (**default:** `http://169.254.170.2/metadata`) |
+| `statsEndpoint` | no | `string` | The URL of the ECS container stats (**default:** `http://169.254.170.2/stats`) |
+| `timeoutSeconds` | no | `integer` | The maximum amount of time to wait for API requests (**default:** `5`) |
+| `labelsToDimensions` | no | `map of string` | A mapping of container label names to dimension names. The corresponding label values will become the dimension value for the mapped name.  E.g. `io.kubernetes.container.name: container_spec_name` would result in a dimension called `container_spec_name` that has the value of the `io.kubernetes.container.name` container label. |
+| `excludedImages` | no | `list of string` | A list of filters of images to exclude.  Supports literals, globs, and regex. |
+
+
+
+
+## Metrics
+
+The following table lists the metrics available for this monitor. Metrics that are not marked as Custom are standard metrics and are monitored by default.
+
+| Name | Type | Custom | Description |
```

Comment:
Talk to @mstumpfx about how to get these in the whitelist properly.  Mark, they are the same as what is emitted by the `docker-container-stats` monitor.

Reply:
hm yeah I can help you with this @seonsfx - are we building new dashboards for this integration?

Reply:
@mstumpfx I understood so. Something very similar with name `ECS Metadata`